### PR TITLE
I18n for schema plugin

### PR DESCRIPTION
**Pull request by dev-samiur**

### CODE REVIEW

File: plugins/schema/components/Person/AttributeForm.js [hunk 1]
```diff
@@ -206,16 +206,16 @@ function AttributeForm({ item, customOnSubmit, hideButtons }) {
...
+                <option value="USER">{t('options.user')}</option>
               </Input>
               <ErrorMessage name="editType">
                 {(msg) => <div style={{ color: 'red' }}>{msg}</div>}
               </ErrorMessage>
             </Col>
           </FormGroup>
           <FormGroup row>
-            <GluuLabel label={t('View Type')} />
+            <GluuLabel label={t('fields.view_type')} />
```
Comment (syntrydy) [inline plugins/schema/components/Person/AttributeForm.js:218]: the GluuLabel component already has the translation built-in.
For example, it should be <GluuLabel label="fields.view_type" />

File: plugins/schema/components/Person/AttributeForm.js [hunk 2]
```diff
@@ -225,13 +225,13 @@ function AttributeForm({ item, customOnSubmit, hideButtons }) {
...
-                <option>{t('ADMIN')}</option>
-                <option>{t('USER')}</option>
+                <option value="ADMIN">{t('options.admin')}</option>
+                <option value="USER">{t('options.user')}</option>
               </Input>
             </Col>
           </FormGroup>
           <FormGroup row>
-            <GluuLabel label={t('Usage Type')} required />
+            <GluuLabel label={t('fields.usage_type')} required />
```
Comment (syntrydy) [inline plugins/schema/components/Person/AttributeForm.js:234]: same here

**Review by syntrydy — CHANGES_REQUESTED** [review]

File: plugins/schema/components/Person/AttributeForm.js (hunk 1 above)
Comment (dev-samiur) [reply plugins/schema/components/Person/AttributeForm.js]: fixed

File: plugins/schema/components/Person/AttributeForm.js (hunk 2 above)
Comment (dev-samiur) [reply plugins/schema/components/Person/AttributeForm.js]: fixed

**Review by syntrydy — APPROVED** [review]

**Pull request merged**